cmd/kubeadm/app/constants/constants.go

```diff
@@ -467,15 +467,11 @@ var (
 
 	// SupportedEtcdVersion lists officially supported etcd versions with corresponding Kubernetes releases
 	SupportedEtcdVersion = map[uint8]string{
-		13: "3.2.24",
-		14: "3.3.10",
-		15: "3.3.10",
-		16: "3.3.17-0",
 		17: "3.4.3-0",
 		18: "3.4.3-0",
-		19: "3.4.13-0",
-		20: "3.4.13-0",
-		21: "3.4.13-0",
+		19: "3.4.18-0",
+		20: "3.4.18-0",
+		21: "3.4.18-0",
```

Comment:
Keep consistent with https://github.com/kubernetes/kubernetes/blob/e1ad9bee5bba8fbe85a6bf6201379ce8b1a611b1/cluster/images/etcd/migrate/options.go#L31

Reply:
I found that this tag does not exist at the moment, maybe someone needs to sync it
`registry.k8s.io/etcd:3.4.18-0`

Reply:
@neolit123 not sure if we need to revert the version map here of change  `3.4.13-0` to `3.4.18-0`.

3.4.18 is the last patch of 3.4.x etcd.

But we did not build it in kube. The latest one is `3.4.13-3`.
https://github.com/kubernetes/kubernetes/blob/release-1.21/cluster/images/etcd/Makefile#L32-L37

```
[root@pacoall ~]# docker pull registry.k8s.io/etcd:3.4.13-4
Trying to pull repository registry.k8s.io/etcd ... 
Pulling repository registry.k8s.io/etcd
Error: image etcd:3.4.13-4 not found
[root@pacoall ~]# docker pull registry.k8s.io/etcd:3.4.13-3
Trying to pull repository registry.k8s.io/etcd ... 
3.4.13-3: Pulling from registry.k8s.io/etcd
5749e56bea71: Pull complete 
99ef90998473: Pull complete 
bc941152f6e4: Pull complete 
011c6af5a966: Pull complete 
2e72bd25dd5b: Pull complete 
Digest: sha256:528ff5e3c80ca0d58d1ab15dc329dc300f9c3fefaffa4c822ca2c389ce254644
Status: Downloaded newer image for registry.k8s.io/etcd:3.4.13-3
```

Reply:
@jpbetz @serathius

https://github.com/kubernetes/kubernetes/blob/1ff1a26426fedbb929bc966cebb9f90995e52725/cluster/images/etcd/Makefile#L29-L40

Should we build all `BUNDLED_ETCD_VERSIONS` in some ways?

Reply:
It does not have `3.4.18-x`. https://github.com/kubernetes/k8s.io/blob/main/registry.k8s.io/images/k8s-staging-etcd/images.yaml#LL5C85-L5C85

May be we should revise to `3.4.13` ?

Reply:
> not sure if we need to revert the version map here of change `3.4.13-0` to `3.4.18-0`.

we have to, if the .18 image does not exist.
only once an image exists, we can use it in k/k code.

Reply:
@xmudrii Would you please look this?

Reply:
@pacoxu i submitted PR https://github.com/kubernetes/kubernetes/pull/118733 for kubeadm